src/commonMain/kotlin/exchange.dydx.abacus/processor/wallet/WalletProcessor.kt

```diff
@@ -121,14 +121,31 @@ internal class WalletProcessor(
         }
     }
 
-    internal fun updateHeight(
+    fun updateHeight(
+        existing: InternalWalletState?,
+        height: BlockAndTime?,
+    ): Triple<InternalWalletState?, Boolean, List<Int>?> {
```

Comment:
should we just make a data type for this? the `Boolean` in particular is not super clear

Reply:
Yeah.. we can do that later... right now it's just using the same return type as the old code.